src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipServiceTest.java

```diff
@@ -117,4 +127,10 @@ private LimitedPartnershipSubmissionDto createDto() {
 
         return submissionDto;
     }
+
+    private Transaction createTransaction() {
+        Transaction transaction = new Transaction();
+        transaction.setId(TRANSACTION_ID);
+        return transaction;
+    }
```

Comment:
This is a duplicate of method buildTransaction()

Reply:
now using buildTransaction which was imported after the merge with @mwestacott code